frontend/packages/dev-console/src/components/topology/topology-types.ts

```diff
@@ -5,7 +5,9 @@ export interface ResourceProps {
   kind: string;
   apiVersion?: string;
   metadata: ObjectMetadata;
-  status: {};
+  status: {
+    [key: string]: any;
```

Comment:
Better type than `any`? Looks like we reference `phase` at the very least.

Reply:
I tried keeping phase here but then we would get errors for other keys, we are referencing other keys in test data and type `any` because other keys might have number, string, object, etc...